### PR TITLE
Add shortcode and initial info for community showcase grid

### PR DESCRIPTION
**Pull request by geealbers**
Here's my first pass at the community showcase. 

I've created a new YAML file (projects.yml) to collect info on each project, using the material we have in figures.yml as a seed for it, but I've only filled out three sample projects thus far. My goal here was to provide some basic info about the project, and then link to the project itself. I used our pop-up system to display some details as a way of not having to fill the page with a lot of text clutter.

### CODE REVIEW

**Comment by geealbers** [comment]
@Erin-Cecele you can see a live preview at https://deploy-preview-287--quire.netlify.app/community/community-showcase/. I'll be interested in your thoughts. The one thing I know already is that I need to think more about the "customization level". Also, there's a bug where you have to click on the word "More" to open the pop-up, clicking on the icon next to it doesn't work for some reason.

Once we get the formatting and info nailed down, I'll ask you and Audrey to take the lead formatting the rest of the projects YAML.

**Comment by Erin-Cecele** [comment]
Great! 

Might be easiest to discuss this on a call, but here are some of my thoughts: 

- It feels a little redundant to have the title on the image itself and then again in the caption. Instead, maybe it could say "View publication" or something to that effect. 
- If we do keep the title, we should switch it so that the title appears above the name of the organization/publication type. And in that case, I don't think the organization name should still be bolded. 
- I'm on the fence about "more information." I'm not sure how necessary that depth of context is. But I do like the inclusion of the Quire user name, so people can reach out to community members directly. That will be very useful. 
- If we do keep the "more information" feature, I would suggest we request our community members give us a one-sentence description of their project when they submit for inclusion on the website. I want to avoid any potential back-and-forth with community members who might not be happy with our descriptions. 
- I agree that we need to think more about "customization level". If we stick with the number system we would need to define it somewhere, I suppose at the top of the page following the description? We'd need to establish a rubric as well (ie 5 or more custom features is a level x, etc.) We could keep it simple with just "quire basic" and "quire customized" and in place of "more information" we could list out what features were customized (again our community members would submit this information to us.)
- Finally, Are you thinking of including a link for each publication on homepage slider as well?

**Comment by geealbers** [comment]
Good feedback, thank you!

I added the "title info" because I found I often can't read it off the images at the size, however, you're right that it's probably not necessary. I'll try it without and with a view publication link instead.

I'd like to try to keep the "more info" in, at least for now, because I think it's an opportunity to give important context about Quire that users of the showcase could benefit from. I agree that it would be ideal to solicit this info from the publishers/users themselves, as long as we frame it as "please give us a sentence or two about your publication and how you used Quire". or whatever. Also, links to things like Minneapolis' write up on building Alcohol's Empire, or to the video with Mills, SJMA and Journal18, might be good to include here. Anything that helps other users evaluate the project's use of Quire and how it might help inform new Quire work of their own.

For "customization" I like your idea of using simple words. I was originally thinking numbers and some sort of graphic, but hadn't figured it out. Will keep thinking/trying things.

More soon ....

**Comment by geealbers** [comment]
@Erin-Cecele I haven't pushed new commit yet, but what do you think about the hierarchy of information here?

I like the View link in place of the title/author, and I like bringing the Quire user name out from the pop-up. Do I have to label them that way? is it too repetitive? Would it make sense without it? And I was questioning the bolding of the publisher/institution name, but I guess when we have them all in there, it will be a nice way for people to scan and see who's using Quire.

Screen Shot 2020-10-07 at 4 03 51 PM

**Comment by Erin-Cecele** [comment]
I really like the way this looks! 

- I like the "View" link rather than hyperlinked title. 
- I also like having Quire User info but let's try it with just the handle. I'm not sure it's self-explanatory without the words "Quire User" but I'd rather keep it less cluttered.
- And I like the bold for publisher/institution
- The only tiny change I'd make is switching "type" and "year" so for Bending Lines it would read "Online Exhibition | 2020".
- I'm just not sure where "customization level" would go. I don't want it to be too wordy. Any new thoughts on that?

**Comment by geealbers** [comment]
@Erin-Cecele, here it is with those changes. not sure those github usernames mean much without some sort of tag. and clicking through to their github page doesn't necessarily explain anything. but I'm on the fence.

Screen Shot 2020-10-08 at 11 29 19 AM

**Comment by Erin-Cecele** [comment]
Yeah, I feel on the fence about it too. Once people start using the forum then user handles might be more intuitive/recognizable. 

That said, we could always try "Contact: @xyz"? It's more succinct than "Quire user".

**Comment by geealbers** [comment]
I left it at "Quire user" for now because I don't think you can really contact someone through GitHub, or that these folks would want to be contacted that way. Down the road I'll also maybe do more with the More Info area and the customization, but this feels fine to start. And I fixed the other things I wanted to fix as well.

@Erin-Cecele if everything looks ok, could either you or Audrey go on this branch and fill out the rest of the `projects.yml` file before we merge it all in? There are instructions for the different fields at the top of the document. Once they're all in, I'm going to take a look at the homepage slider. I'd rather populate that with info from the projects.yml file than with duplicated info in figures.yml if possible

Thanks.

**Comment by Erin-Cecele** [comment]
Looks good!! And I like what you did with the submission button at the bottom.

I think I'll have Audrey start it and then I will go in and complete any missing fields. This will be a good opportunity for Audrey to get better acquainted with our users and their projects. 

Thanks for all your work on this.
